Product_Landing_Page/public/index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Landing Page</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" href="../css/style.css">
+  <link rel="stylesheet" type="text/css" href="../css/font-awesome.min.css">
+</head>
+<body>
+  <div class="container" id="container">
+  <header class="header" id="header">
+    <div class="page-header" id="page-header">
+
+    </div>
```

Comment:
If you not using this div.Please remove this.

Reply:
Done